packages/lib-classifier/src/components/Classifier/components/QuickTalk/QuickTalk.js

```diff
@@ -176,18 +101,18 @@ function QuickTalk ({
           )}
           <Anchor
             alignSelf='center'
-            label={t('QuickTalk.aria.goToTalk')}
+            label={t('QuickTalk.goToTalk')}
             href={subject.talkURL}
-            icon={<FormNextLink size='small' />}
+            icon={<ShareRounded size='small' />}
             margin='xsmall'
             target='_blank'
             rel='nofollow noopener noreferrer'
             reverse={true}
             size='xsmall'
           />
```

Comment:
Accessibility issue: the link text for this link is 'Or, view this Subject Discussion on Talk ShareRounded'. I think it's meant to be something like 'Or, view this Subject Discussion on Talk, (opens in a new window/tab)'.

<img width="770" alt="Screenshot of the link text in the accessibility tree." src="https://user-images.githubusercontent.com/59547/180407978-385e299c-5fb2-4481-b1b2-5b57784a1f96.png">

Reply:
This should probably be caught by the tests.

Reply:
This is an annoying problem because it won't be caught by the accessibility checks in the storybook, since the link does have text. Libraries that set meaningless alt text for icons would be better off setting no alt text at all, to be honest.

Reply:
I'll try to add a proper "(opens in a new window/tab)" alt to that icon and update the queries accordingly. One sec.

Reply:
👍 Added "opens in new window/tab" aria-label (a11yTitle since it's a Grommet component) + translations in [bd9a309](https://github.com/zooniverse/front-end-monorepo/pull/3413/commits/bd9a309b2ea15fd9ac96adb76a42b838ebcf02a9)